manim/mobject/geometry.py

```diff
@@ -1554,3 +1556,254 @@ def __init__(self, main_shape, *mobjects, **kwargs):
+        A sequence of two :class:`int` numbers determining which of the 4 quadrants should be used.
+        Possibilities: (1,1), (-1,1), (1,-1), (-1,-1).
+    other_angle : :class:`bool`
+        Toggles between the two possible angles defined by two points and an arc center. If set to
+        False (default), the arc will always go counterclockwise from the point on line1 until
+        the point on line2 is reached. If set to True, the angle will go clockwise from line1 to line2.
+    dot : :class:`bool`
+        Allows for a :class:`Dot` in the arc. Mainly used as an convention to indicate a right angle.
+        The dot can be customized in the next three parameters.
+    dot_radius : :class:`float`
+        The radius of the :class:`Dot`.
+    dot_distance : :class:`float`
+        Placement of the dot in the arc. This distance from the arc center is given as the radius
+        diveded by this :class:`float` parameter.
+    dot_color : :class:`Colors`
+        The color of the :class:`Dot`.
+    kwargs
+        Further keyword arguments that are passed to the constructor of :class:`Arc`.
+
+    Examples
+    --------
+    The first example shows some right angles with a dot in the middle while the second example shows
+    all 8 possible angles defined by two intersecting lines.
+
+    .. manim:: RightArcAngleExample
+
+        class RightArcAngleExample(Scene):
+            def construct(self):
+                line1 = Line( LEFT, RIGHT )
+                line2 = Line( DOWN, UP )
+                rightarcangles = [
+                    ArcAngle(line1, line2, dot=True),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1), dot=True, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8, dot=True, dot_color=YELLOW, dot_radius=0.05, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, dot=True, dot_color=GREEN, dot_radius=0.07),
+                ]
+                line_list = VGroup( *[VGroup() for k in range(4)] )
+                for k in range(4):
+                    linea = line1.copy()
+                    lineb = line2.copy()
+                    line_list[k].add( linea )
+                    line_list[k].add( lineb )
+                    line_list[k].add( rightarcangles[k] )
+                line_list.arrange_in_grid(buff=1.5)
+                self.play(
+                    *[ShowCreation(
+                        line_list[k][:2]
+                        ) for k in range(4)]
+                )
+                self.wait(0.5)
+                self.play(
+                    LaggedStart(
+                        *[ShowCreation(
+                            line_list[k][2]
+                        ) for k in range(4)],
+                        lag_ratio=0.7
+                    )
+                )
+                self.wait()
+
+    ..manim:: ArcAngleExample
+
+        class Angles_Test3(Scene):
```

Comment:
The class name don't match and so this fails to render in[ your PR's documentation](https://manimce--1085.org.readthedocs.build/en/1085/reference/manim.mobject.geometry.ArcAngle.html#manim.mobject.geometry.ArcAngle). Also, this should be made an image.
```suggestion
    .. manim:: Angles_Test3
        :save_last_frame:

        class Angles_Test3(Scene):
```